### PR TITLE
Refractor update bookmark by

### PR DESCRIPTION
**Pull request by schkiho**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,25 +4,26 @@ const mongoose = require("mongoose")
 
 const PORT = process.env.PORT || 4000
 
-const indexRouter = require("./routes/index")
+//const indexRouter = require("./routes/index")
```
Comment (Bilie) [inline app.js]: Is this line still needed? If not, we can delete it

File: app.js [hunk 2]
```diff
@@ -4,25 +4,26 @@ const mongoose = require("mongoose")
...
+  })
+  .catch(err => {
+    console.error("Could not connect, ", err)
+  })
 
 app.use(logger("dev"))
 app.use(express.json())
 
-app.use("/", indexRouter)
+// app.use("/", indexRouter)
```
Comment (Bilie) [inline app.js]: Hi, the same question here

File: controller/bookmark.js [hunk 3]
```diff
@@ -53,34 +49,63 @@ module.exports = {
...
-      const updatedBookmark = db
-        .get("bookmarks")
-        .find({ id })
-        .assign({ url }, { tags })
-        .write()
-
-      res.locals.response = Object.assign({}, res.locals.response || {}, {
-        updatedBookmark
+    const updateBookmark = req.body
+    console.log(req.params)
```
Comment (Bilie) [inline controller/bookmark.js]: Hi, logs should not be committed

File: controller/bookmark.js [hunk 4]
```diff
@@ -53,34 +49,63 @@ module.exports = {
...
+      })
+      .finally(() => {
+        next()
       })
-    }
-
-    next()
   },
 
+  /*====================================================*
```
Comment (Bilie) [inline controller/bookmark.js]: Hi, it looks like this part also should not have been committed.

File: controller/bookmark.js [hunk 5]
```diff
@@ -53,34 +49,63 @@ module.exports = {
...
+        .send("The bookmark with the given ID was not found.")
+
+    Bookmark.updateOne({ _id: id }, updateBookmark, { new: true })
+      .then(updatedBookmark => {
+        res.locals.response = Object.assign({}, res.locals.response || {}, {
+          bookmark: updatedBookmark
+        })
+      })
+      .catch(err => {
+        console.error(err)
```
Comment (Bilie) [inline controller/bookmark.js]: Hi, the user should get the error status and message when the update fails

**Comment by ctocco** [comment]
everything looks great will merge now.

**Comment by ctocco** [comment]
resolving issue #32 case closed

**Pull request merged**